README.md

```diff
@@ -1852,7 +1862,27 @@ interface. Valid value is an integer.
 
 ##### `switchport_autostate_exclude`
 Exclude this port for the SVI link calculation. Valid values are 'true', 'false', and 'default'. This property is not supported on IOS XR.
-
+##### `private_vlan_mapping`
```

Comment:
The verbage in this README section is a bit confusing in my opinion.  I would suggest the following general updates.
- Specify `vlans` instead of  `vlan` when referring to one or more vlans.
- Not sure what you mean by `vlans associated on this port`.  Is this more accurate? `private vlans associated with the primary vlan`
- Instead of `Valid values are integers` I think it would be better to indicate `Valid values are an array of ["primary_vlan", "secondary_vlan"] pairs`